overlord/devicestate/handlers_install.go

```diff
@@ -320,5 +320,16 @@ func checkEncryption(model *asserts.Model) (res bool, err error) {
 		return false, nil
 	}
 
+	// encryption is available but some models may prefer to not
+	// use it
+	// TODO: provide way to select via install chooser menu
+	// if the install is unencrypted or encrypted
+	if model.StorageSafety() == asserts.StorageSafetyPreferUnencrypted {
+		logger.Noticef(`installing system unencrypted because of "storage-safety: prefer-unencrypted"`)
```

Comment:
"installing system unencrypted following|respecting prefer-unencrypted setting for storage-safety model option"

naive speakers might have some input on this

Reply:
hmm I'd say just
```
installing system unencrypted due to prefer-unencrypted storage-safety model option
```

Reply:
@anonymouse64 thanks that sounds less clunky than both my proposal and the original

Reply:
I'd maybe change _due_ in Ian's suggestion as it sounds a little as if the setting may have been unintentionally set (slightly). Words like heed or obey may be more definitive. I think I'd go with comply with: `installing system unencrypted to comply with prefer-unencrypted storage-safety model option`